httpclient5/src/test/java/org/apache/hc/client5/http/auth/TestCredentials.java

```diff
@@ -84,9 +84,9 @@ public void testUsernamePasswordCredentialsHashCode() {
         final UsernamePasswordCredentials creds3 = new UsernamePasswordCredentials(
                 "name", "otherpwd".toCharArray());
 
-        Assert.assertTrue(creds1.hashCode() == creds1.hashCode());
+        Assert.assertEquals(creds1.hashCode(), creds1.hashCode());
         Assert.assertTrue(creds1.hashCode() != creds2.hashCode());
```

Comment:
Why didn#t you modify this one?

Reply:
I think it's more clear. There is a specific assertions for compare the two object

Reply:
`assertNotEquals()` isn't clear?

Reply:
Ok, reverted.
TY

Reply:
I am confused now. I have reverted the first line but left the third one? It is now even more inconsistent. My question was: Why can the second line use `assertNotEquals()`?

Reply:
Sorry. My mistake. I leave it as it was